server/index.js

```diff
@@ -0,0 +1,39 @@
+const express = require('express');
+const bodyParser = require('body-parser');
+const db = require('../db/index');
+let app = express();
+
+app.use(express.static(__dirname + '/../client/dist'));
+app.use(bodyParser.json());
+app.use(bodyParser.urlencoded({extended: true}));
+
+app.get('/rooms/similarListings', (req, res) => {
```

Comment:
the route should be /rooms/:roomid/similarListings
as we agreed for all our components

Reply:
Let me fix this now and push.

Reply:
Issue Fixed @rafaelhernandez